WebUI/lims/views.py

```diff
@@ -11,10 +11,23 @@
 from django.views.decorators.http import require_http_methods
 from django.contrib import messages
 from lims.shared import *
+from psycopg2 import sql
```

Comment:
Is there a reason why we are importing psycopg2 as well as using the from keyword? Seems like it should be one or the other?